package.json

```diff
@@ -25,12 +29,16 @@
   },
   "homepage": "https://github.com/hoodiehq/hoodie-client#readme",
   "devDependencies": {
+    "browserify": "^12.0.1",
     "istanbul": "^0.3.20",
     "istanbul-coveralls": "^1.0.3",
+    "npm-run-all": "^1.3.0",
+    "semantic-release": "^4.3.5",
     "standard": "^5.3.0",
     "tap-spec": "^4.1.0",
     "tape": "^4.2.0",
-    "semantic-release": "^4.3.5"
+    "uglify-js": "^2.6.1",
+    "uglifyify": "^3.0.1"
```

Comment:
I wanted to use just uglifyify, but they recommend using both uglifyify on individual files and uglifying the output of that again. https://github.com/hughsk/uglifyify